doc/about.rst

```diff
@@ -96,86 +96,339 @@ following paper:
 Artwork
 -------
 
-High quality PNG and SVG logos are available in the `doc/logos/ <https://github.com/scikit-learn/scikit-learn/tree/master/doc/logos>`_ source directory.
+High quality PNG and SVG logos are available in the `doc/logos/
+<https://github.com/scikit-learn/scikit-learn/tree/master/doc/logos>`_
+source directory.
 
 .. image:: images/scikit-learn-logo-notext.png
    :align: center
 
 Funding
 -------
+Scikit-Learn is a community driven project, however institutional and private
+grants help to assure its sustainability.
+
+The project would like to thank the following funders. 
+
+...................................
+
+.. raw:: html
+
+   <div class="sk-sponsor-div">
+   <div class="sk-sponsor-div-box">
+
+.. figure:: images/png-logo-inria-la-fondation.png
+   :width: 100pt
+   :target: https://scikit-learn.fondation-inria.fr/en/home/
+
+   The `Scikit-Learn Consortium at Inria Foundation
+   <https://scikit-learn.fondation-inria.fr/en/home/>`_ `Members 
+   <https://scikit-learn.fondation-inria.fr/en/home/#sponsors>`_ fund Olivier
```

Comment:
```suggestion
   <https://scikit-learn.fondation-inria.fr/en/home/#sponsors>`_ funds Olivier
```

Reply:
May I suggest to rephrase as "The members of the Scikit-Learn Consortium at Inria Foundation fund...".

This would be easier to read and also visually highlights the fact that there are two links.

Reply:
done in aa269a0e886115a2da6dd77aee1993510bdfef22